lib/utils/index.ts

```diff
@@ -0,0 +1,5 @@
+// use theses type checking helpers to avoid mistyping "undefind", I mean "undfined"
+export const isUndefined = value => typeof value === "undefined";
+export const isString = value => typeof value === "string";
+export const isNumber = value => typeof value === "number";
+export const isFunction = value => typeof value === "function";
```

Comment:
I could have used lodash for this but it seemed like too much code: (eg. https://github.com/lodash/lodash/blob/master/isFunction.js)